float64 goal_pose_y
float64 pose_y
float64 goal_pose_x
float64 pose_x

---
float64 result
